### ISSUE TITLE
Synchronize cookies consent between Richie and FUN Apps

### ISSUE THREAD

**Issue by jbpenrath** [openfun/fun-apps#712]
## Description
Currently on FUN Mooc, our visitors have to accept/refuse tracking on Richie and then on OpenEdX. This situation could be confusing for some people which do not distinguish the two platforms.

Furthermore, on OpenEdX, cookie banner offers only the choice to accept cookie. Until the user clicks on "Accept" button, tracking is disabled but cookie banner is always shown.

## Proposal
First on OpenEdX, we should add a button "Refuse" to hide the cookie banner and do not track user.
Then as OpenEdX and Richie are on the same root domain, we could try to use the same cookie name on both platform. Thus cookie consent should be share between the two platforms.

**Comment by sampaccoud** [comment]
+ change cookie consent to tarteaucitron.js as well?

**Comment by jbpenrath** [comment]
Yes I think it is a good idea to homogenize our cookie consent solutions and cookie sharing should be implicit.
